media_service/models.py

```diff
@@ -138,6 +138,10 @@ class Meta:
     def __unicode__(self):
         return u'{0}:{1}:{2}'.format(self.id, self.lti_context_id, self.title)
 
+
+def metadata_default():
```

Comment:
(it might make [the unit test](https://github.com/Harvard-ATG/media_management_api/pull/25/files#diff-4b60429b247c00428f1e210c6cde90edR64) a little more intuitive, for instance)